README.md

```diff
@@ -36,3 +36,23 @@ address to our [Slack Team invite page](https://slack-join-intel-media.herokuapp
 
 Slack complements our other means of communication.  Pick the one that works
 best for you!
+
+# How to install Libva
+```
+git clone https://github.com/intel/libva.git
+cd libva
+./autogen.sh
+make
+sudo make install
+```
+
+# How to install Libva with Meson (http://mesonbuild.com)
+```
+meson build
+ninja -C build
+```
+
+# How to add Libva package configuration to environment on CentOS
+```
+export PKG_CONFIG_PATH=${PKG_CONFIG_PATH}:/usr/local/lib/pkgconfig
```

Comment:
why this CentOS specific has to be in libva?  PKG_CONFIG_PATH should be set for those that compile using prefix, system variable is sufficient.

I believe this should not be merged.

Reply:
System variable was not enough for us, but I agree with your comment. 
Thanks!